File: server/src/main/java/gg/mineral/server/entity/object/Minecart.java
package gg.mineral.server.entity.object;

import gg.mineral.server.entity.EntityImpl;
import gg.mineral.server.world.WorldImpl;

public class Minecart extends EntityImpl {

    public Minecart(int id, WorldImpl world) {
        super(id, world);
    }

    @Override
    public void tickAsync() {
    }

}

File: server/src/main/kotlin/gg/mineral/server/entity/object/Minecart.kt
package gg.mineral.server.entity.`object`

import gg.mineral.server.entity.EntityImpl
import gg.mineral.server.snapshot.AsyncServerSnapshotImpl
import gg.mineral.server.world.WorldImpl

class Minecart(id: Int, serverSnapshot: AsyncServerSnapshotImpl, world: WorldImpl) :
    EntityImpl(id, serverSnapshot, world)
